### DISCUSSION TITLE
Version 6.2.4 Release Candidate 1

### DISCUSSION THREAD

**Discussion by jrincayc** [jrincayc/ucblogo-code#166] [General]
## What's Changed
* ISSUE-150: Validate RANDOM range input has the smaller number first by @dmalec in https://github.com/jrincayc/ucblogo-code/pull/151
* Manual updates by @jrincayc in https://github.com/jrincayc/ucblogo-code/pull/152
* Avoid more implicit function declarations by @fweimer-rh in https://github.com/jrincayc/ucblogo-code/pull/156
* Move wait include to else of windows. by @jrincayc in https://github.com/jrincayc/ucblogo-code/pull/157
* ISSUE-158: Configure wxWidgets to avoid linking to brew installed libraries by @dmalec in https://github.com/jrincayc/ucblogo-code/pull/159
* Migrate to wxWidgets 3.2 by @dmalec in https://github.com/jrincayc/ucblogo-code/pull/163
* Configure wxWidgets OSX build to avoid linking to brew installed regex library by @dmalec in https://github.com/jrincayc/ucblogo-code/pull/164

## New Contributors
* @fweimer-rh made their first contribution in https://github.com/jrincayc/ucblogo-code/pull/156

**Full Changelog**: https://github.com/jrincayc/ucblogo-code/compare/version_6.2.3...version_6.2.4rc1

<hr /><em>This discussion was created from the release <a href='https://github.com/jrincayc/ucblogo-code/releases/tag/version_6.2.4rc1'>Version 6.2.4 Release Candidate 1</a>.</em>

**Comment by dmalec** [comment]
I tried [ucblogo_624rc1.dmg.zip](https://github.com/jrincayc/ucblogo-code/releases/download/version_6.2.4rc1/ucblogo_624rc1.dmg.zip) on both:
1. OSX 10.15.7 on Intel Core i7 - primary dev machine so it has various homebrew installed libraries
2. OSX 13.2.1 on Apple M2 - mostly stock machine, has XCode and built from source versions of autoconf, automake, & m4 (to try and avoid false positives due to homebrew)

Worked well on both

**Reply by jrincayc** [reply]
Thanks for testing.  I checked the windows version and the linux tar file, so I will probably make the 6.2.4 release tonight.

**Reply by dmalec** [reply]
Sounds good, please let me know if there's anything I can help with.

**Reply by jrincayc** [reply]
I made the 6.2.4 release.
